tests/test_state_estimators.py

```diff
@@ -45,13 +47,21 @@ def test_state_est_template(self):
 
     def __test_state_est(self, StateEstimatorClass, ModelClass):
         m = ModelClass(process_noise=5e-2, measurement_noise=5e-2)
+        # where we initialize, put various other data types
+        x_scalar = ScalarData({'x': 1.75, 'v': 35})
+
         x_guess = {'x': 1.75, 'v': 35} # Guess of initial state, actual is {'x': 1.83, 'v': 40}
         x = m.initialize()
+        # dict initial state is x_guess
 
-        filt = StateEstimatorClass(m, x_guess)
-        x_guess = filt.x.mean  # Might be new
+        filt = StateEstimatorClass(m, x_guess) # passed into state estimator function, test state estimator in same way
+        filt_kf_scalar = KalmanFilter(m, x_scalar)
```

Comment:
I think we want to take these out of the general __test_state_est method and instead have these in the specific test_UKF/test_KF methods